README.md

```diff
@@ -1,102 +1,64 @@
 
 These vectors are packed into a dictionty D and passed to train (found in train.py module) as demonstrated in Run.py.
 
-
-
-
-
-
-
-
-
+The provided example scripts assume the data is a .mat file containinig, 'Survival', 'Censored', and either 'Integ\_X' or 'Gene\_X' depending on what feature set we are using.
 
 ## Installation Guide for Docker Image
 
+A Docker image for SurvivalNet is provided for those who prefer not to build from source. This image contains an installation of SurvivalNet on a bare Ubuntu operating system along with sample data used in our paper published in *Scientific Reports*. This helps users avoid installation of the */bayesopt/* package and other dependencies required by SurvivalNet.
 
-This project was build on Docker Platform for the easy use and the platform independency. The link for Docker Image is found [here](https://hub.docker.com/r/cramraj8/survivalnet2.0/).
-You can pull the Docker Image using this command on terminal.
+The SurvivalNet Docker Image can either be downloaded [here](https://hub.docker.com/r/cramraj8/survivalnet2.0/), or can be pulled from Docker hub using the following command:
     
     sudo docker pull cramraj8/survivalnet2.0
 
-
-Docker Image (***cramraj8/survivalnet2.0***) was built on top of Ubuntu-Docker Image. All the dependencies and libraries was added into the Docker Image. The *Bayesian Optimization* Python package was already installed inside the Docker Image. This ***Bayesian Optimization(BayesOpt package)*** can be located by */bayesopt/* folder.
-
-The survivalNet python package will be found inside the *Ubuntu-Docker* along with *BayesOpt* folder. 
-
-
-(Download and) Run the Docker Image (from Docker Hub) on local machine
+Running this image on your local machine with the command
     
     sudo docker run -it cramraj8/survivalnet2.0 /bin/bash
 
+launches a terminal within the image where users have access to the package installation. 
 
-This command will look for the **survivalnet2.0** Docker Image locally and if not found, then the Docker Engine will look at the Docker Hub.
-Once the Download is completed, a Docker Container will be created, and the terminal will turn into bash mode.
-
-
-
-Now you are inside the Docker Container.
-The project package is located inside the */SurvivalNet/* folder. 
-
-If you want to run the example python script, you can navigate into
+Example python scripts can be found in the folder 
     
     cd /SurvivalNet/examples/ 
-folder.
-There you can find some python scripts. But *Run.py* is the proper python script that can tell you about the deep learning progress.
+
+where users can see how to train and validate deep survival models. The main script
     
     python Run.py
```

Comment:
@cramraj8 Is Run.py in the examples folder? Are there other files in the examples folder?